client-course-schedulizer/src/components/reuseables/Schedule/Schedule.scss

```diff
@@ -39,15 +43,14 @@ $header-offset: 8;
 .time-slot {
   display: flex;
   justify-content: flex-end;
-  margin-top: -1.5 * $spacing;
+  height: 50px; // TODO: Calculate or use constant?
```

Comment:
On my computer, there's something that is covering the bottom half of the click area for the buttons on the toolbar. I'm not sure if it is this or not

Reply:
#76